kubernetes-dev-environment.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: dev-environment
-          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp:dev-environment
+        - name: allennlp-dev-env
+          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp/allennlp-gpu:latest
```

Comment:
I was going to say "point at the dockerhub image" but the point of this change I guess is "make some changes" -> "push your own image" -> "run that image on your k8s cluster"

I don't think we want randos *pushing* to our ecr repo :)

ideally it would be configurable?